taxi/tests/test_views.py

```diff
@@ -0,0 +1,124 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from django.urls import reverse
+
+from taxi.models import Car, Manufacturer
+
+URLS = {
+    "taxi:index": reverse("taxi:index"),
+    "taxi:manufacturer-list": reverse("taxi:manufacturer-list"),
+    "taxi:driver-list": reverse("taxi:driver-list"),
+    "taxi:car-list": reverse("taxi:car-list")
+}
+
+
+class UnauthorizedAccessTests(TestCase):
+    def test_login_required(self):
+        for name, url in URLS.items():
+            response = self.client.get(url)
+            self.assertNotEqual(
+                response.status_code,
+                200,
+                msg=f"Error accessing URL ({name}): "
+                    f"{url} returned {response.status_code}"
+            )
+
+
+class AuthorizedAccessTests(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="user",
+            password="testuser",
+            license_number="ABC12345"
```

Comment:
Ensure that the `Driver` model includes a `license_number` field, as it is being used in this test case.